api/address/routes.go

```diff
@@ -52,6 +54,18 @@ type esdtTokenData struct {
 	Properties      string `json:"properties"`
 }
 
+type esdtNFTTokenData struct {
+	TokenIdentifier string   `json:"tokenIdentifier"`
+	Balance         string   `json:"balance"`
+	Properties      string   `json:"properties"`
+	Name            string   `json:"name"`
+	Creator         string   `json:"creator"`
+	Royalties       string   `json:"royalties"`
+	Hash            []byte   `json:"hash"`
```

Comment:
Why `Hash` field is not a string?  
Ask this question because everywhere on the API we return hash (hash of a block or hash of a transaction, etc)  as string hex-encoded.

Reply:
in case of address codehash is returned as byte array.